docs/howto/how-to-cross-build-runtime-for-aarch64.md

```diff
@@ -1,2 +1,102 @@
-# How to Cross-build Runtime for Aarch64
+# How to Cross-build Runtime for AARCH64
 
+In **ONE**, we use `AARCH64` on build files such as Makefile, CMakeLists.txt and so on.
+
+## Prepare AARCH64 Ubuntu RootFS
+
+Install required packages
+
+```
+$ sudo apt-get install qemu qemu-user-static binfmt-support debootstrap
+```
+
+Use `build_rootfs.sh` script to prepare Root File System. You should have `sudo`
+
+```
+$ sudo ./tools/cross/build_rootfs.sh aarch64
+```
+- supports `arm`(default) and `aarch64` architecutre for now
+- supports `xenial`(default) and `trusty` release
+
+To see the options,
+```
+$ ./tools/cross/build_rootfs.sh -h
+```
+
+RootFS will be prepared at `tools/cross/rootfs/aarch64` folder.
+
+***\* CAUTION: The OS version of rootfs must match the OS version of execution target device. On the other hand, you need to match the Ubuntu version of the development PC with the Ubuntu version of rootfs to be used for cross-build. Otherwise, unexpected build errors may occur.***
+
+If you are using Ubuntu 16.04 LTS, select `xenial`, if you are using Ubuntu 18.04 LTS, select `bionic`. You can check your Ubuntu code name in the following way.
+
+```
+$ cat /etc/lsb-release
+DISTRIB_ID=Ubuntu
+DISTRIB_RELEASE=18.04
+DISTRIB_CODENAME=bionic
+DISTRIB_DESCRIPTION="Ubuntu 18.04.4 LTS"
+```
+
+If a build error occurs because the version of the development system and the target system do not match, and if you can't replace your development system for any reason, you can consider [cross-build using the docker image](how-to-build-runtime-using-prebuilt-docker-image.md).
+
+### Prepare RootFS at alternative folder
+
+Use `ROOTFS_DIR` to a full path to prepare at alternative path.
+
+```
+$ ROOTFS_DIR=/home/user/rootfs/aarch64-xenial sudo -E ./tools/cross/build_rootfs.sh aarch64
+```
+
+### Using proxy
+
+If you need to use proxy server while building the rootfs, use `--setproxy` option.
+
+```
+# for example,
+$ sudo ./tools/cross/build_rootfs.sh aarch64 --setproxy="1.2.3.4:8080"
+# or
+$ sudo ./tools/cross/build_rootfs.sh aarch64 --setproxy="proxy.server.com:8888"
+```
+
+This will put `apt` proxy settings in `rootfs/etc/apt/apt.conf.d/90proxy` file
+for `http`, `https` and `ftp` protocol.
+
+## Cross build for AARCH64
+
+Install cross compilers
+```
+$ sudo apt-get install gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
+```
+
+Build and install ARM Compute Library
+```
+$ CROSS_BUILD=1 TARGET_ARCH=aarch64 make acl
+```
+Mostly you only need once of ACL build. This will build and install to
+`Product/(target_arch-os)/out/bin` folder.
+- this is required for `AARCH64` on Ubuntu
```

Comment:
This is outdated.  Please remove this.

Reply:
@hseok-oh , the section removed. Thanks!